content/posts/expiring-auth-tokens.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Breaking Changes to Ecosounds & A2O Authentication"
+date: 2025-03-14T14:20:00+10:00
+image: "/images/ecosounds-banner-image.jpg"
+imageCredit: "QUT Ecoacoustics Staff"
+tags:
+    - News
+    - Technical
+---
+
+The Acoustic Workbench software that powers sites such as [Ecosounds](https://www.ecosounds.org/) and
+[A2O](https://data.acousticobservatory.org/) has been upgraded with enhanced security features.
+These changes will break existing workflows and automated scripts that depend on authentication token.
+
+## Effected Users
+
+If you are using the Ecosounds or the A2O through the website interface, you will not be effected by these changes.
+
+These authentication changes will effect the following (but not limited to) users who run automated scripts such as:
+
+- R scripts
+- Powershell scripts
+- HPC scripts
+- Automated workflow scripts
+
+Effected users may not experience the breaking changes immediately, as they are likely to arise when scripts run across
```

Comment:
Just nix this sentence, it's already happened

Reply:
Done.

Just for reference, this wasn't supposed to be a comment as to when we're rolling out the changes.

It was supposed to be a sentance explaining that although you might not see your scripts break immediately because your auth token might not rotate while your script is running.

What I don't want people to do is see that their auth token works during their 9-5 workday, get confident and then become confused (and irritated) when it breaks in a multi-day HPC job.

---

The information is later duplicated in the "Expiring Authentication Tokens" section